### DISCUSSION TITLE
Exporting Descriptor Wallet from Bitcoin Core

### DISCUSSION THREAD

**Discussion by BTC4life69** [ordinals/ord#1520] [Q&A]
Hiiiii. Is there a way to export a descriptor wallet in Bitcoin core? Apparently you can’t dump private keys. Also the descriptor wallet checkbox  is checked by default when creating a wallet so I unknowingly created one rip

**Comment by casey** [comment]
Hiiiiiiiii. Copy-pasted from this great answer by Andrew Chow on Bitcoin Talk:

> Descriptor wallets do not allow dumpprivkey because the fundamental principle behind descriptor wallets is that private keys are not enough information to transport a wallet. Private keys lack derivation information and lack information about what kind of scripts to create. They also do not work for wallets that have anything more complicated than just single key scripts. Thus allowing a RPC that only outputs private keys would be working against the point of having descriptors.Instead of dumpprivkey, descriptor wallets have listdescriptors. This will output all of the descriptors stored in the wallet, which means that it will include information about derivation paths and scripts to create. Descriptors are a full backup of the key and script information stored in the wallet. With 23.0, listdescriptors will also be able to optionally output descriptors containing private keys.

I think you want `bitcoin-cli listdescriptors true` to print the descriptors, with the `true` argument making it include the private keys in the descriptor.

**Reply by casey** [reply]
Also, if this worked, smash that "mark as answer" button. I live for those green check marks.

**Reply by BTC4life69** [reply]
Hey this ends up giving long list repeating “descriptors” “time stamp” “active” “ range” and “next” then repeating. Don’t see any mention of private keys and unsure how I would import the wallet into a new pc from there

**Reply by BTC4life69** [reply]
https://www.reddit.com/r/Bitcoin/comments/zv39gy/how_to_import_descriptor_wallet_bitcoin_cli/ seems like a relevant thread but unsure what conclusion it came to

**Reply by BTC4life69** [reply]
maybe importdesciptors and then paste "desc" strings .... just weird because everyone talks about private keys but now its seems like bitcoin core is trying to push this descriptor wallet that uses a different name/method ?

**Reply by BTC4life69** [reply]
where/how do you run "bitcoin-cli listdescriptors true"

**Reply by nick07002** [reply]
this is what I was looking for. good answer and references. ty
